checker/checker_dartagnan.go

```diff
@@ -51,16 +53,36 @@ var models = map[MemoryModel]struct {
 	VMM:   {"vmm.cat", "c11"},
 }
 
-func (c *DartagnanChecker) run(ctx context.Context, testFn string) (string, error) {
+func catFilePath(mm MemoryModel) string {
 	dartagnanHome := tools.GetEnv("DARTAGNAN_HOME")
 
+	modelInfo, has := models[mm]
+	if !has {
+		logger.Fatalf("could not find info for model '%v'", mm)
+	}
+
+	if b := tools.GetEnv("DARTAGNAN_CAT_PATH"); b != "" {
+		if cpath := filepath.Join(b, modelInfo.cat); tools.FileExists(cpath) == nil {
+			return cpath
+		}
+	}
+
+	cpath := filepath.Join(dartagnanHome, "cat", modelInfo.cat)
+	if err := tools.FileExists(cpath); err != nil {
+		logger.Fatalf("could not find custom cat file: %v", err)
+	}
+	return cpath
+}
+
+func (c *DartagnanChecker) run(ctx context.Context, testFn string) (string, error) {
+
 	opts := []string{
-		"--property=program_spec,liveness",
+		"--property=program_spec,cat_spec,liveness",
```

Comment:
@db7 just for you to know, this change has no effect because the default solver is caat which ignores data races.

However, I still think it is good to have the property here and then if the user wants to enable data race detection, it only needs to set the `--method=assume` option rather than also having to pass the whole list of properties

Reply:
That was the idea! Thanks for confirming it.